.github/workflows/test.yml

```diff
@@ -69,16 +77,19 @@ jobs:
         echo ::set-output name=test-env::$ENV
 
     - name: Initialize Postgres DB
+      if: steps.skiptest.outputs.is-bump == 'no'
       env:
         PGPASSWORD: postgres
       run: psql -h 127.0.0.1 -U postgres -f ./service/local-dev/local-postgres-init.sql
 
     - name: Set up AdoptOpenJDK 11
+      if: steps.skiptest.outputs.is-bump == 'no'
       uses: joschi/setup-jdk@v2
       with:
         java-version: 11
 
     - name: Cache Gradle packages
+      if: steps.skiptest.outputs.is-bump == 'no'
```

Comment:
It's amazing they don't support yaml anchors for this kind of thing https://github.community/t/support-for-yaml-anchors/16128/10